app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from "react-redux";
+import { fetchMessage } from "../redux/messages/messages";
+
+const Greeting = () => {
+  const message = useSelector((state) => state.messagesReducer);
+  const dispatch = useDispatch();
+  
+  useEffect(() => {
+    dispatch(fetchMessage());
+  }, []);
+
+  return (
```

Comment:
- [ ] The App has all the files but the greeting messages are not randomly displayed like you said.

Reply:
![screenshot](https://user-images.githubusercontent.com/51248936/154107050-d4743cb2-ed1e-4deb-b0a1-bcc3fb0a7bdb.png)

Reply:
Great ask for review

Reply:
Here is the screenshot. If you want I can make a Loom video to show you how it works.

Reply:
No need you can ask for review now